down_camera: 0
forward_camera: 1

line_threshold: 161

HSV_min_yellow1: 0
HSV_min_yellow2: 0
HSV_min_yellow3: 0
HSV_max_yellow1: 180
HSV_max_yellow2: 255
HSV_max_yellow3: 255

BGR_min_yellow1: 0
BGR_min_yellow2: 0
BGR_min_yellow3: 0
BGR_max_yellow1: 255
BGR_max_yellow2: 255
BGR_max_yellow3: 255

HSV_min_red1: 0
HSV_min_red2: 0
HSV_min_red3: 0
HSV_max_red1: 180
HSV_max_red2: 255
HSV_max_red3: 255

BGR_min_red1: 0
BGR_min_red2: 0
BGR_min_red3: 0
BGR_max_red1: 255
BGR_max_red2: 255
BGR_max_red3: 255

HSV_min_blueH1: 70
HSV_min_blueH2: 0
HSV_min_blueH3: 0
HSV_max_blueH1: 180
HSV_max_blueH2: 255
HSV_max_blueH3: 60

BGR_min_blueH1: 70
BGR_min_blueH2: 0
BGR_min_blueH3: 0
BGR_max_blueH1: 255
BGR_max_blueH2: 255
BGR_max_blueH3: 60

HSV_min_redX1: 0
HSV_min_redX2: 0
HSV_min_redX3: 110
HSV_max_redX1: 75
HSV_max_redX2: 75
HSV_max_redX3: 255

BGR_min_redX1: 0
BGR_min_redX2: 0
BGR_min_redX3: 110
BGR_max_redX1: 75
BGR_max_redx2: 75
BGR_max_redX3: 255
